src/index.js

```diff
@@ -1,27 +1,38 @@
 import 'bootstrap/dist/css/bootstrap.css'
 
+import { Container, Navbar, NavbarBrand } from 'reactstrap'
+import { css, cx } from 'react-emotion'
 import { faCompass } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { Container, Navbar, NavbarBrand } from 'reactstrap'
-import { injectGlobal } from 'emotion'
+
 import React from 'react'
 import ReactDOM from 'react-dom'
 
+import { GRAY_900 } from './assets/colors'
 import FeedbackButton from './components/feedback-button'
 import MortgageInvestmentCompare from './components/mortgage-investment-compare/index'
+import { DARK_BLUE_GRAY } from './assets/colors'
 
-// TODO 2019-09-15: Where should global styles live?
-injectGlobal`
-  b {
-    font-weight: 500;
-  }
-`
+const footer = cx(
+  css(`background-color: ${DARK_BLUE_GRAY};`),
```

Comment:
preferred to have it here as opposed to having all of this inline, but I'm still trying to stick to bootstrap classes so that the syntax is a little more familiar to you. I assume we'll eventually move it to its own Footer component but we can probably leave this here while it's small.